.gitignore

```diff
@@ -1,3 +1,6 @@
+# VSCode
+.vscode/*
```

Comment:
I like this take, will keep it in mind in the future. I decided to remove my workspace config in favor of switching it to user since it was an insignificant setting.